css/style.css

```diff
@@ -0,0 +1,2929 @@
+      margin-left: 0;
+      margin-top: 5px;
+      padding-left: 50px;
+  }
+
+  .bottom__appraisal-title {
+      padding-left: 0;
+  }
+
+  .bottom__content {
+      width: 292px;
+      padding-left: 0;
+      margin-left: 0;
+  }
+    
+  .stars {
+      padding-right: 0;
+  }
+    
+  .bottom__shield {
+      display: none;
+  }
+
+  .bottom__btn {
+      margin-left: 152px;
+  }
+
+  .button__btn {
+      margin-top: 0;
+      margin-left: 0;
+  }
+
+  .bottom__info-img {
+      display: none;
+   }  
+
+}
+
+@media (max-width: 850px) {
+  .container {
+      max-width: 100%;
+  }
+
+  .top__wrapper-charset {
+      margin-right: 0;
+  }
+
+  .charset__speed {
+      margin-left: 100px;
+  }
+
+  .top__wrapper-item {
+      margin-right: 110px;
+  }
+
+  .button__outline {
+      padding-left: 0;
+      padding-right: 20px;
+  }
+
+  .button__red {
+      padding-left: 25px;
+      padding-right: 25px;
+      margin-right: 0;
+  }
+
+  .middle__inner {
+      max-width: 850px;
+  }
+
+  .middle__fon {
+      max-width: 850px;
+  }
+
+  .middle__info-row {
+      max-width: 850px;
+  }
+
+  .middle__row {
+      margin-right: 0;
+  }
+
+}
+
+@media (max-width: 800px) {
+  .container {
+    max-width: 100%;
+  }
+
+  .top {
+      text-align: center;
+      background: url(../img/tesla-fon.png)  no-repeat;
+      background-position: 50% 0;
+      background-size: 800px 467px;
+  }
+
+ .top__wrapper {
+      display: flex;
+      flex-direction: column;
+      margin-left: 10px;
+      margin-top: 37px;
+  }
+
+  .top__wrapper-charset {
+      background: url(../img/svg/road-ipad.svg) no-repeat;
+      background-position: 420px 0;
+  }
+
+  .top__wrapper-item {
+      margin: 30px 0px;
+      display: flex;
+      justify-content: center;
+      margin-top: 45px;
+      margin-bottom: 60px;
+      margin-left: 15px;
+  }
+
+  .charset {
+      margin: 0px 30px;
+      justify-content: center;
+  }
+
+  .charset__speed {
+      width: 65.92px;
+      height: 50.16px;
+      padding-left: 0;
+      margin-left: 92px;
+  }
+
+  .charset__road {
+      display: none;
+      width: 37.83px;
+      height: 60.72px;
+      margin-left: 20px;
+  }
+
+  .charset__first-big {
+      font-weight: 600;
+      font-size: 50px;
+      padding-left: 15px;
+      padding-bottom: 5px;
+  }
+
+  .charset__first-small {
+      margin-right: 62px;
+      padding-right: 0px;
+      padding-left: 7px;
+      padding-bottom: 10px;
+  }
+
+  .button {
+      margin: 0;
+      justify-content: center;
+  }
+
+  .button__outline {
+      color: #e51937;
+      padding-bottom: 10px;
+      padding-top: 16px;
+      padding-right: 40px;
+      padding-left: 45px;
+      background: url(../img/svg/steering-ipad.svg) no-repeat;
+      background-position: 0 6px;
+  }
+
+  .button__outline-img {
+      display: none;
+      padding-left: 28px;
+      margin-right: 15px;
+      margin-left: 0;
+  }
+
+  .button__text {
+      width: 110px;
+      font-weight: 700;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/vovafinal/pullRequest?prid=5998857)